projects/ngx-formentry/src/form-entry/value-adapters/obs-adapter-helper.ts

```diff
@@ -326,11 +326,11 @@ export class ObsAdapterHelper {
   // PAYLOAD GENERATION FUNCTIONS
   addFieldNameSpaceandPath(node, obs) {
     obs.formFieldNamespace = this.formFieldNamespace;
-    obs.formFieldPath = `${node?.question?.questionIndex}${node?.nodeIndex}${this.obsIndex}`;
+    obs.formFieldPath = `${this.obsIndex}~${node?.question?.extras?.id}`;
```

Comment:
Is obsIndex really useful outside of this context, i.e., can we successfully calculate it from just the form schema?